app/models/va_health_facility.rb

```diff
@@ -1,115 +1,116 @@
+# frozen_string_literal: true
 require_dependency 'facilities/client'
 
 class VAHealthFacility < ActiveModelSerializers::Model
-  attr_accessor :id, :station_number, :visn_id, :name, :classification, :lat, :long, 
+  attr_accessor :station_id, :station_number, :visn_id, :name, :classification, :lat, :long,
                 :address, :phone, :hours, :services
 
   def self.query(bbox:, services:)
     results = client.query(bbox: bbox, where: VAHealthFacility.where_clause(services))
-    results.each_with_object([]) do | record, facs |
+    results.each_with_object([]) do |record, facs|
       facs << VAHealthFacility.from_gis(record)
     end
   end
 
   def self.find_by_id(id:)
     results = client.get(identifier: id)
-    VAHealthFacility.from_gis(results.first) unless results.nil?
+    VAHealthFacility.from_gis(results.first) unless results.nil? || results.empty?
```

Comment:
`blank?` is a Rails method that will combine these two 
http://api.rubyonrails.org/classes/Object.html#method-i-blank-3F